app/geo/gui/src/ts/twxplore/gui/geo/attributeStrategies/FrequencyFeatureAttribute.ts

```diff
@@ -1,13 +1,15 @@
 import {FeatureAttribute} from "./FeatureAttribute";
 import {FeatureAttributeName} from "../states/map/FeatureAttributeName";
 import {FilterType} from "../states/map/FilterType";
+import {TypeOfFeatureAttribute} from "../states/map/TypeOfFeatureAttribute";
+import {FieldType} from "../states/map/FieldType";
 
 export class FrequencyFeatureAttribute implements FeatureAttribute {
   static readonly instance = new FrequencyFeatureAttribute();
 
-  readonly isNumeric = true;
   readonly name = FeatureAttributeName.transmissionPower;
-  readonly isString = false;
+  readonly typeOf = TypeOfFeatureAttribute.NUMBER;
```

Comment:
Confusing why you have this and dataType/fieldType. Just combine them? Either way, typeOf is way too generic a name.